gazebo_plugins/include/gazebo_plugins/gazebo_ros_force.hpp

```diff
@@ -66,6 +69,10 @@ class GazeboRosForce : public gazebo::ModelPlugin
   /// Optional callback to be called at every simulation iteration.
   virtual void OnUpdate();
 
+  /// Optional callback to be called at every simulation iteration when the force
+  /// is applied on the body frame.
+  virtual void OnUpdateRelative();
```

Comment:
Sorry @alexfneves , I just realized that this can't go into dashing because adding `virtual` functions is not guaranteed not to break ABI. Can you change the implementation to use `OnUpdate` on both cases? I think you may need to bring back the `force_on_world_frame_` member variable.